### ISSUE TITLE
Since 3.0.3, styling for attachment container border always grey

### ISSUE THREAD

**Issue by ppawel** [mattermost/platform#3254]
#### Summary
We just upgraded to 3.0.3 and our TeamCity notifications lost color.

Suspected problem is those CSS rules:

![image](https://cloud.githubusercontent.com/assets/469990/15799540/77451386-2a60-11e6-95d4-211f53d6adfd.png)

#### Expected behavior
In 2.2, it looked like this:

![image](https://cloud.githubusercontent.com/assets/469990/15799562/1954fbdc-2a61-11e6-9ed9-76f671aa21a0.png)

It was green for build success :)

#### Observed behavior

Now it always looks like this:

![image](https://cloud.githubusercontent.com/assets/469990/15799544/9a5cd0ac-2a60-11e6-8128-64f10b5d78d2.png)

**Comment by lfbrock** [comment]
Hi @ppawel, 

How are you setting the border color in your attachment? Can you post an example of the JSON for the attachment? 

I think this may already be fixed in v3.1, but if you post an example we can test it out to confirm.

**Comment by ppawel** [comment]
```
{ "channel_id" : "mp7x5dks8tbfif8j39j7nnnsnh",
  "create_at" : 1464796855132,
  "delete_at" : 0,
  "filenames" : [  ],
  "hashtags" : "",
  "id" : "6ot496xcxjbb9qycio1t1rqewr",
  "message" : "[Failed - ....",
  "original_id" : "",
  "parent_id" : "",
  "pending_post_id" : "",
  "props" : { "attachments" : [ { "color" : "danger",
            "fallback" : "Tests",
            "fields" : [ { "isShort" : false,
                  "title" : "Tests",
                  "value" : "asdasd"
                }
              ]
          } ],
      "from_webhook" : "true",
      "override_icon_url" : "https://raw.githubusercontent.com/PeteGoo/tcSlackBuildNotifier/master/docs/TeamCity72x72.png",
      "override_username" : "TeamCity"
    },
  "root_id" : "",
  "type" : "slack_attachment",
  "update_at" : 1464799824694,
  "user_id" : "uf3o6c5i8p8yirfri1mict148e"
}
```

**Comment by lfbrock** [comment]
Hi @ppawel - I think I know what the problem is, it looks like the "danger"/"good"/"warning" color isn't working. Can you try switching it to a hex value for the color and see if that works?

**Comment by ppawel** [comment]
No, I can't (easily) do that - it's sent by TeamCity Slack notifier plugin, I would need to edit the source code for that plugin...

It worked in MM 2.2 and there was no change with the plugin on TeamCity side so it looks like a regression in Mattermost...

**Comment by lfbrock** [comment]
Alright it's fine, I'm fairly sure that's what the issue is so I created a ticket here to track it: https://mattermost.atlassian.net/browse/PLT-3273

We've already cut the release candidate for v3.1, so this probably won't get fixed until v3.2. 

If anyone from the community wants to make a contribution for this, we would welcome a pull request!

**Comment by ppawel** [comment]
OK, thanks for the info, looking forward to the release.

**Comment by GavBurke** [comment]
Commenting on this as I have 3.0.0 but cannot see a fix in the release logs between 3.0.0 and 3.5, other than PLT-3273 closed as cannot reproduce.

Anyways, the issue is div settings of

`<div data-class=".app__body .attachment .attachment__container1"><style>.app__body .attachment .attachment__container {border-left-color:#1B2C3E}</style></div>`

Override the red/green colours of 

`.attachment .attachment__container.attachment__container--good {
    border-left-color: #00c100;
}`

So your CSS is messed up since v3 dropped

**Comment by ppawel** [comment]
This was fixed with upgrading to 3.1, not sure why it's not in the change log or in JIRA though.
